addons/sale/views/payment_views.xml

```diff
@@ -1,13 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
 
-    <record id="acquirer_form_inherit_sale" model="ir.ui.view">
-        <field name="name">payment.acquirer.form.inherit.sale.payment</field>
-        <field name="model">payment.acquirer</field>
-        <field name="inherit_id" ref="payment.payment_acquirer_form"/>
+    <record id="provider_form_inherit_sale" model="ir.ui.view">
+        <field name="name">payment.provider.form.inherit.sale.payment</field>
```

Comment:
```suggestion
    <record id="payment_provider_form" model="ir.ui.view">
        <field name="name">payment.provider.form.inherit.sale</field>
```

Drop the `inherit_sale` from the id, the module is automatically added in xml ids anyway.
Let's have `payment_provider_form` as id for all payment provider form views :)

Reply:
Don't forget this comment too